packages/pluginsdk-peerdeps/sync-deck.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+/* @ts-check */
+/* eslint-disable no-console */
+/* Synchronizes versions of all (non-dev) dependencies from Deck's root package.json */
```

Comment:
After a plugin is scaffolded, this script copies the versions from deck's `package.json`, but only for regular dependencies.  This is used to keep the version of `react` up to date, for example.